dangerfile.js

```diff
@@ -31,6 +33,31 @@ if ( ! pr.body.includes( 'Proposed changelog entry' ) ) {
 		'"Proposed changelog entry" is missing for this PR. Please include any meaningful changes'
 	);
 }
+
+// Check if newly added .php files were added to phpcs linter whitelist
+if ( newFiles.length > 0 ) {
+	const newPHPFiles = danger.git.created_files.filter(
+		fileName => fileName.includes( '.php' ) && ! fileName.includes( 'tests/php' )
+	);
+
+	const notWhitelistedFiles = [];
+
+	newPHPFiles.forEach( file => {
+		const whitelistedPath = phpWhitelist.find( path => file.includes( path ) );
+		if ( ! whitelistedPath ) {
+			notWhitelistedFiles.push( file );
+		}
+	} );
+
+	if ( notWhitelistedFiles.length > 0 ) {
+		const stringifiedFilesList = '\n' + notWhitelistedFiles.join( '\n' );
+		warn(
+			'Consider adding new PHP files to PHPCS whitelist for automatic linting:' +
```

Comment:
Let's reference where the whitelist is located.
```suggestion
			'Consider adding new PHP files to PHPCS whitelist (`bin/phpcs-whitelist.js`) for automatic linting:' +
```